src/mlpack/core/data/scaler_methods/scaling_model.hpp

```diff
@@ -0,0 +1,140 @@
+/**
+ * @file scaling_model.hpp
```

Comment:
Do you think this file would be more appropriate in `src/mlpack/methods/preprocess/` since that's where it's used?

Reply:
Yes :)